### PR TITLE
Custom mapbox server

### PR DESCRIPTION
**Pull request by sumarlidason**
enables configuration of the mapbox endpoint

ref https://github.com/mapbox/kepler.gl/issues/1

### CODE REVIEW

File: src/mapbox/mapbox.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ const propTypes = {
   gl: PropTypes.object, /** External WebGLContext to use */
 
   mapboxApiAccessToken: PropTypes.string, /** Mapbox API access token for Mapbox tiles/styles. */
+  mapboxApiUrl: PropTypes.string,
```
Comment (Pessimistress) [inline src/mapbox/mapbox.js]: Add this to the `StaticMap` and `InteractiveMap` components documentation as well?

Comment (sumarlidason) [reply src/mapbox/mapbox.js]: @Pessimistress thanks for 👀; updated 👍

File: src/mapbox/mapbox.js [hunk 2]
```diff
@@ -346,6 +350,7 @@ export default class Mapbox {
 
     // Creation only props
     this.mapboxgl.accessToken = props.mapboxApiAccessToken || defaultProps.mapboxApiAccessToken;
+    this.mapboxgl.config.API_URL = props.mapboxApiUrl;
```
Comment (xintongxia) [inline src/mapbox/mapbox.js]: there is another prop `baseApiUrl` documented in [mapbox-gl-js](https://docs.mapbox.com/mapbox-gl-js/api/#baseapiurl), do you know the difference between `baseApiUrl` and `this.mapboxgl.config.API_URL`?

Comment (sumarlidason) [reply src/mapbox/mapbox.js]: @xintongxia you are right, that method has been available since v0.52. I'll update it.

**Pull request merged**